packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.tsx

```diff
@@ -81,7 +81,16 @@ function QuadraticGraph(props: QuadraticGraphProps) {
             aria-label={srQuadraticGraph}
             aria-describedby={`${quadraticDirectionId} ${quadraticVertexId} ${quadraticInterceptsId}`}
         >
-            <Plot.OfX y={y} color={color.blue} />
+            <Plot.OfX
+                y={y}
+                color={color.blue}
+                svgPathProps={{
+                    // Use aria-hidden to hide the line from screen readers
```

Comment:
Ooo fun I never looked into this but I'm glad mafs allowed us to pass props through.